qt/src/OutputEditorHOCR.cc

```diff
@@ -1223,11 +1225,18 @@ void OutputEditorHOCR::savePDF() {
 	if(!failed.isEmpty()) {
 		QMessageBox::warning(m_widget, _("Errors occurred"), _("The following pages could not be rendered:\n%1").arg(failed.join("\n")));
 	}
+
+    bool pdfCanBeOpened = true;
 	try {
 		document->Close();
 	} catch(PoDoFo::PdfError& e) {
+        pdfCanBeOpened = false;
 		QMessageBox::warning(m_widget, _("Export failed"), _("The PDF export failed (%1).").arg(e.what()));
 	}
+    if(m_pdfExportDialogUi.checkBoxOpenOutputPdf->isChecked() && pdfCanBeOpened) {
+        QDesktopServices::openUrl(QUrl("file:///" + outname));
```

Comment:
You should probably urlencode the outname.

Reply:
For what? all works fine now. Do you know better way here?

Reply:
Per RFC 3986 non all characters are valid in a URI and if your filename contains special characters (even just spaces), opening might fail. See QUrl::toPercentEncoding

Reply:
I found good solution: QDesktopServices::openUrl(QUrl::fromLocalFile(file));

Reply:
Great!